conans/client/rest/conan_requester.py

```diff
@@ -7,6 +7,7 @@
 
 import requests
 from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.ssl_ import create_urllib3_context
```

Comment:
Please, move this `import` inside the `init_poolmanager`, it will fail for `requests` `2.16.0`, `2.16.1` and `2.16.2`.